organization/views.py

```diff
@@ -266,8 +281,11 @@ def get_success_url(self):
         else:
             return reverse('view-organization', kwargs={'slug': organization_id})
 
+    def get_cancel_url(self):
+        return reverse('view-organization', kwargs={'slug': self.kwargs['organization_id']})
```

Comment:
Yes I noticed that when implementing these. I agree we should unify the approaches. Which one do you think we should go with, `slug` or `organization_id`?

Reply:
`organization_id` is a more accurate name since the variable refers to the UUID assigned to the organization, but I believe `slug` is used more widely across the app. There's a tradeoff involved here, but ultimately I think `organization_id` is easier to understand than `slug`, in spite of the wide reach of `slug`.